package.json

```diff
@@ -44,7 +44,7 @@
     "heimdalljs-logger": "^0.1.7",
     "json-stable-stringify": "^1.0.0",
     "rsvp": "^4.8.2",
-    "workerpool": "^2.3.0"
+    "workerpool": "^6.1.4"
```

Comment:
So I wasn't intending to drop support for any node version 🤔  the idea was that the tests should still check the correct Node version. I guess technically because workerpool aren't testing against our supported Node versions they could break older ones at any time, is that what you mean?

Reply:
> I guess technically because workerpool aren't testing against our supported Node versions they could break older ones at any time, is that what you mean?

Yes, exactly. They could update to (for example) async/await syntax, which would be breaking for Node 8 users (which ember-cli-babel@6 still supports).